components/gitpod-db/src/typeorm/generate-latest-migration.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationExecutor } from "typeorm";
+import { writeFile } from "node:fs/promises";
+import { Config } from "../config";
+import { TypeORM } from "./typeorm";
+import { join } from "node:path";
+
+async function generateLatestMigrationName() {
+    const config = new Config();
+    const typeorm = new TypeORM(config, {});
+    const conn = await typeorm.getConnection();
+    const t = await new MigrationExecutor(conn).getAllMigrations();
+    await writeFile(join(__dirname, "latest-migration.txt"), t.sort((m) => m.timestamp).pop()?.name ?? "");
```

Comment:
This will require a db to be started. Not sure it is worth it. Mybe it's simpler to just look at the files?